robottelo/constants.py

```diff
@@ -21,6 +21,113 @@
     'CLOSED'
 ]
 
+CUST_PERMS_LIST = [
```

Comment:
@san7ket any reason why not implement this straight away? it should be really easy to implement that and would save us an extra PR.
I simply see no reason in merging something we will be changing straight away.